src/app/destinyTrackerApi/reviewModesFetcher.ts

```diff
@@ -0,0 +1,36 @@
+import { t } from 'i18next';
+import { D2ManifestDefinitions } from "../destiny2/d2-definitions.service";
+
+export interface D2ReviewMode {
+  mode: number;
+  description: string;
+}
+
+export enum ActivityModeHashes {
+  playerVersusEnemy = 1164760493,
+  playerVersusPlayer = 1164760504,
+  raid = 2043403989,
+  trials = 1370326378
+}
+
+export enum DtrActivityModes {
+  notSpecified = 0,
+  playerVersusEnemy = 7,
+  playerVersusPlayer = 5,
+  raid = 4,
+  trials = 39
```

Comment:
You can express these in terms of their enums, like: `trials = DestinyActivityModeType.TrialsOfTheNine`. You may not even need this.

Reply:
I went with the enumerated type so that we're not dealing with magic numbers internally for the agreed-upon mode numbers that DTR's OK with receiving (and we're expecting).

Internally, we can point to `DestinyActivityModeType` for most, but this falls down on `notSpecified` (there's a few modes where the type is 0 in the manifest), and with the data that we're receiving from the API, we don't get a mode (but we could certainly perform a lookup and post-process an object in).

If it would make more sense, I could move this over to the tracker service, so I could have `DtrUserReview`'s `mode` property of this type (and tighten the interfaces on methods in there from accepting `mode` as a `number` to accepting it as a `DtrActivityMode`.